### ISSUE TITLE
Adding multiple drives

### ISSUE THREAD

**Issue by clemone210** [ivan-pinatti/docker-torrent-box-with-vpn#9]
Hi,
I currently have 1 20TB Share mounted to the /media folder.
How is it possible to add another share to be used by the containers?

**Comment by ivan-pinatti** [comment]
Hi @clemone210,

You could change the folder mapping/mounting at the OS level and preserve the path for the Docker containers.

More info at; [https://linuxize.com/post/how-to-mount-and-unmount-file-systems-in-linux/](https://linuxize.com/post/how-to-mount-and-unmount-file-systems-in-linux/)

Example, you wanna have your share for movies, you could simply mount whatever folder you have to that specific path, `/media/Movies`

**Comment by clemone210** [comment]
Thanks for the reply.
Do you think it would be somehow possible to mount both shares to /mnt/share1 and /mnt/share2 and link them to ./media?

Currently only share1 has the folder structure ./media/movies ./media/series ./media/anime and so on.

Share2 is currently empty.
I dont want to move 10TB series to share2 as both shares are externally hosted.

Basically I would like to have the folder structure on both shares and both should be mapped to the ./media folder.

**Comment by ivan-pinatti** [comment]
Check if this helps [https://www.networkinghowtos.com/howto/combine-several-mount-points-into-a-single-virtual-one-using-mhddfs/](https://www.networkinghowtos.com/howto/combine-several-mount-points-into-a-single-virtual-one-using-mhddfs/)

**Comment by clemone210** [comment]
Okay I thinks should work.
Thanks!

Another question.
I have [overseerr](https://github.com/sct/overseerr) setup as a separate docker container.
How can I connect to radarr / sonarr trough api?

I use cloudflared to expose the services to the web (zero trust).
It works for all the containers by using 127.0.0.1:port as hostname/ip-address.

When I want to connect to radarr from overseer, I cannot get a connection.
Do you may know why?
The server is a Ubuntu Root Server with direct WAN acces with static IPV4. 
So basically I do not have a CIDR to exclude into protonvpn .env file.
Maybe this is the issue?

**Comment by ivan-pinatti** [comment]
Try running it in host mode, then you can try connecting using the 127... network.

What you also could do, is to add the overseer config to the Docker Compose file.

**Comment by ivan-pinatti** [comment]
In addition, I don't know what your use case is, but, you could simply create a public list on IMDB or another service and sync Sonarr / Radarr to fetch from the list. Would that help?
